test/parallel/test-fs-rmdir-recursive.js

```diff
@@ -7,6 +7,14 @@ const fs = require('fs');
 const path = require('path');
 const { validateRmdirOptions } = require('internal/fs/utils');
 
+const realEmitWarning = process.emitWarning;
+process.emitWarning = () => {
+  // Reset back to default after the test.
+  process.emitWarning = realEmitWarning;
+
+  throw new Error('deprecated');
+};
+
```

Comment:
We do have a helper to check for deprecation messages. Please check `expectWarning` from the common test utils. There's also a documentation for it in the common README.

Reply:
That's great. I was hoping there was a better way to do this.